armi/nucDirectory/nuclideBases.py

```diff
@@ -440,6 +460,46 @@ def factory(force=False):
         __readMc2Nuclides()
         _completeNaturalNuclideBases()
         elements.deriveNaturalWeights()
+        riplDataPath = _findRiplData()
+        if riplDataPath is not None:
+            from armi.nuclearDataIO import ripl
+
+            ripl.makeDecayConstantTable(directory=riplDataPath)
+
+
+def _findRiplData():
+    """
+    Validates that the ``ARMI_RIPL_PATH`` environment variable has been set
+    and points to a valid directory for reading in the RIPL database files.
+    """
+    global RIPL_PATH
+
+    riplEnvironVariable = "ARMI_RIPL_PATH"
+    if riplEnvironVariable not in os.environ:
+        return None
```

Comment:
You can still use the `get(..., None)` idiom, if you want. The problem from before is that we were doing `.get(..., "")` and passing that directly to `Path()`, which led to a `Path` pointing to the cwd. So:
```
riplEnvPath = os.environ.get("ARMI_RIPL_PATH", None)
if riplEnvPath is None:
    return None
path = pathlib.Path(riplEnvPath)
# etcetera
```